src/style.css

```diff
@@ -1 +1,48 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin:;
+}
```

Comment:
Remember to be consistent with your margins according to the checklist. Add only top or bottom margins, but not both, to avoid potential margin collapse issues.